docs/source/faiss_es.mdx

```diff
@@ -32,7 +32,7 @@ FAISS retrieves documents based on the similarity of their vector representation
 >>> ds_with_embeddings.add_faiss_index(column='embeddings')
 ```
 
-4. Now you can query your dataset with the `embeddings` index. Load the DPR Question Encoder, and search for a question with [`Dataset.get_nearest_examples`]:
+4. Now you can query your dataset with the `embeddings` index. Load the DPR Question Encoder, and search for a question with [`~datasets.search.IndexableMixin.get_nearest_examples`]:
```

Comment:
I think you can point to the Dataset method here no ? Or the doc builder is not able to resolve it ?
Same for the other search and tf methods

```suggestion
4. Now you can query your dataset with the `embeddings` index. Load the DPR Question Encoder, and search for a question with [`~datasets.Dataset.get_nearest_examples`]:
```

Reply:
It looks like neither format works even though the function - `get_nearest_examples` -  is in the list of `[[autodoc]] datasets.Dataset` and the [path](https://github.com/huggingface/datasets/blob/32e0e79aa57b36a269b6f2ee38449a043a393e50/src/datasets/search.py#L700) to it is correct.

Reply:
I merged master into this branch to update the doc builder app. Hopefully it fixes the issue

Reply:
Pinging @sgugger for help as well! 🙌

Reply:
Are you sure you have all dependencies installed?

Reply:
We could still change to [`Dataset.get_nearest_examples`] which should work, but not sure why [`~datasets.Dataset.get_nearest_examples`] doesn't work

Reply:
> Are you sure you have all dependencies installed?

I asked myself the same question, but for some reason linking to Dataset.add_faiss_index() does work, even if it's the same dependencies

Reply:
Could you maybe clarify what "doesn't work"? From a quick debugging, doc_builder is able to find the object from the two references (Dataset.get_nearest_examples and datasets.Dataset.get_nearest_examples).

Reply:
Yeah, the functions are only displayed in code format instead of linking to the API docs.

Reply:
Looked into it more, and yes, you will need to link to `datasets.search.IndexableMixin.get_nearest_examples` because that's where the method actually is. Basically the `doc-builder` finds that when looking at `"Dataset.get_nearest_examples"` so it's the anchor name it gives to that method (you can check it's actually in the link to it: https://huggingface.co/docs/datasets/master/en/package_reference/main_classes#datasets.search.IndexableMixin.get_nearest_examples)

I can work on adding more references that point to it so `Dataset.get_nearest_examples` would also work as a reference, but not sure I will have time before next week :-)

Reply:
Disentangled the mess in [this PR](https://github.com/huggingface/doc-builder/pull/163). Once it's merged, you should have intra-links that work better for those kinds of methods :-)

Reply:
Cool thanks ! I guess it means can have [`Dataset.get_nearest_examples`] and merge this PR :)

Reply:
Yes, you can even check it works on the doc build of this PR beforehand since the PR is merged.
On my tests locally, this particular reference was properly resolved.

Reply:
Woo everything looks good now, thanks so much Sylvain! 🥇